crawler/crawler.rb

```diff
@@ -90,11 +82,12 @@
           @log.info "Missed records.."
         end
 
-        StatHat.new.ez_count('Github Events', new_events.size)
+        # StatHat.new.ez_count('Github Events', new_events.size)
 
       rescue Exception => e
         @log.error "Processing exception: #{e}, #{e.backtrace.first(5)}"
         @log.error "Response: #{req.response_header}, #{req.response}"
+        raise "Stop"
```

Comment:
I don't think you want to re-raise here. We don't want to abort the process if we hit an error on a batch: log it, continue.

Reply:
Sorry this was debug code.